.gitlab-ci.yml

```diff
@@ -209,6 +212,35 @@ test-vector:
   script:
     - ./test/dockertest.sh --image "registry.ddbuild.io/images/mirror/${IMAGE}" --script install_script_vector0.sh --vector "true"
 
+e2e_ubuntu_install:
+  stage: e2e
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/test-infra-definitions/runner:$TEST_INFRA_DEFINITIONS_BUILDIMAGES
+  tags: ["arch:amd64"]
+  before_script:
+    # Setup AWS Credentials
+    - mkdir -p ~/.aws
+    - aws ssm get-parameter --region us-east-1 --name ci.agent-linux-install-script.agent-qa-profile --with-decryption --query "Parameter.Value" --out text >> ~/.aws/config
+    - export AWS_PROFILE=agent-qa-ci
```

Comment:
nit: wouldn't this be more suited in the `variables:` section?

Reply:
Actually no, I need to export it after downloading the profile from `dd-build` at previous step. I then activate the profile and from there execute commands in `agent-qa` aws exec

Reply:
Ah ok got it! Thanks for the clarification :)